pattern-library/sass/components/_forms.scss

```diff
@@ -287,7 +287,12 @@
         position: absolute;
         @include right(rem(($baseline/4)));
         margin-top: rem(-($baseline/10)); // vertically centers the down caret icon
-        fill: $input-control-color;
+        color: $input-control-color;
+        vertical-align: middle;
+    }
+
+    .icon-fallback-glyph .icon:before {
+        content: "\25BE"; // system glyph down arrow
```

Comment:
Nice touch! Any way we can make this even easier for folks who need to manually add fall backs and create Sass vars for known/managed icons that we're falling back specifically? If so, I could see us using a var like ``icon-caret-down-fallback`` and then calling it here.